cross/libvpx/patches/libvpx-1.6.1-gcc-no-warn-flags.patch

```diff
@@ -0,0 +1,13 @@
+--- libs.mk.orig	2017-02-14 21:37:49.108060382 +0100
++++ libs.mk	2017-02-14 19:01:20.491912108 +0100
+@@ -117,8 +117,10 @@
+ $(BUILD_PFX)third_party/googletest/%.cc.o: CXXFLAGS += -Wno-missing-field-initializers
+ # Suppress -Wextra warnings in first party code pending investigation.
+ # https://bugs.chromium.org/p/webm/issues/detail?id=1069
++ifndef SPKSRC_GCC_NO_WARN_FLAGS
```

Comment:
To be honest, I don't mind if these couple of lines were just removed in this case. Either the compile process doesn't produce warnings (for current toolchains), or you silence them (for old toolchains)...which means you're not interested in seeing the warnings either way. Well, just turn them off altogether then, right?

That cleans up this PR quite a bit. If the patch adds a comment what the original reason was to turn off `-Wno-unknown-warning-option` and `-Wno-clobbered`, then that's all we need imo.